packages/pds-ember/addon/components/pds/app/drawer/index.hbs

```diff
@@ -0,0 +1,9 @@
+<div
+  class="pds-app__drawer pds-drawer"
```

Comment:
The class names here will be refactored when we get around to defining official Drawer styles. For now, this retains existing behavior.